sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpConnectionScope.cs

```diff
@@ -638,14 +636,15 @@ public void Dispose()
                     linkSettings.DesiredCapabilities.Add(AmqpProperty.EnableIdempotentPublishing);
                 }
 
-                // These values must either all be specified or none.  It is valid for an individual value to be null to signal the
-                // service to supply the value, so long as the key is present.
+                // The property keys on the link must either all be specified or none.  It is valid for an individual option to be missing, which
```

Comment:
@christothes:  Updated this to address your feedback in the previous PR.   Does this better explain?